README.md

```diff
@@ -2893,13 +2893,13 @@ Other Style Guides
     // in some other file
     // bad
     import CheckBox from './checkBox'; // PascalCase import/export, camelCase filename
-    import FortyTwo from './FortyTwo'; // PascalCase import/filename, camelCase export
-    import InsideDirectory from './InsideDirectory'; // PascalCase import/filename, camelCase export
+    import FortyTwo from './fortyTwo'; // PascalCase import/filename, camelCase export
```

Comment:
the comment says the import *and* the filename are PascalCase; so `FortyTwo` is correct

Reply:
Thank you for explanation. I am really sorry about wasting your time!

Reply:
No problem. Thanks for the contribution!